internal/api/jwks.go

```diff
@@ -0,0 +1,29 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	jwk "github.com/lestrrat-go/jwx/v2/jwk"
+)
+
+type JwksResponse struct {
+	Keys []jwk.Key `json:"keys"`
+}
+
+func (a *API) Jwks(w http.ResponseWriter, r *http.Request) error {
```

Comment:
Mind sending a `Cache-Control` header here?

Reply:
made it identical to the one used for returning the SAML metadata